gens/commands/load.py

```diff
@@ -139,10 +140,16 @@ def sample(
     required=True,
     help="Genome build",
 )
-@with_appcontext
-def annotations(file: str, genome_build: GenomeBuild):
+@click.option(
+    "-h",
+    "--header",
+    "has_header",
+    is_flag=True,
+    help="If bed file contains a header",
```

Comment:
Maybe a bit more explicit here

```suggestion
    help="Requires a specific set of headers for bed file. Allowed headers are A, B, C ...",
```

Reply:
I think it is a good idea to allow an header option. But, if we disagree with the exact headers used by Stockholm we have the chance to change the names now. I don't think they mind renaming their headers.

Reply:
I think we should hold of on doing that. Its better to change everything in one go later, makes it less complicated for them

Reply:
Yes, makes sense!